Packs/AzureCompute/Integrations/AzureCompute_v2/test_data/get_vm_command.json

```diff
@@ -0,0 +1,185 @@
+                            "displayStatus": "Provisioning succeeded",
+                            "level": "Info",
+                            "time": "2023-12-21T00:00:00.9991605+00:00"
+                        }
+                    ]
+                }
+            ],
+            "extensions": [
+                {
+                    "name": "enablevmAccess",
+                    "statuses": [
+                        {
+                            "code": "ProvisioningState/succeeded",
+                            "displayStatus": "Provisioning succeeded",
+                            "level": "Info",
+                            "message": "Enable succeeded."
+                        }
+                    ],
+                    "type": "Microsoft.OSTCExtensions.VMAccessForLinux",
+                    "typeHandlerVersion": "1.5.18"
+                }
+            ],
+            "hyperVGeneration": "V2",
+            "osName": "ubuntu",
+            "osVersion": "20.04",
+            "statuses": [
+                {
+                    "code": "ProvisioningState/succeeded",
+                    "displayStatus": "Provisioning succeeded",
+                    "level": "Info",
+                    "time": "2024-08-20T00:00:00.6656328+00:00"
+                },
+                {
+                    "code": "PowerState/running",
+                    "displayStatus": "VM running",
+                    "level": "Info"
+                }
+            ],
+            "vmAgent": {
+                "extensionHandlers": [
+                    {
+                        "status": {
+                            "code": "ProvisioningState/succeeded",
+                            "displayStatus": "Ready",
+                            "level": "Info",
+                            "message": "Plugin enabled"
+                        },
+                        "type": "Microsoft.OSTCExtensions.VMAccessForLinux",
+                        "typeHandlerVersion": "1.5.18"
+                    },
+                    {
+                        "status": {
+                            "code": "ProvisioningState/succeeded",
+                            "displayStatus": "Ready",
+                            "level": "Info",
+                            "message": "Plugin enabled"
+                        },
+                        "type": "Microsoft.CPlat.Core.VMApplicationManagerLinux",
+                        "typeHandlerVersion": "1.0.13"
+                    }
+                ],
+                "statuses": [
+                    {
+                        "code": "ProvisioningState/succeeded",
+                        "displayStatus": "Ready",
+                        "level": "Info",
+                        "message": "Guest Agent is running",
+                        "time": "2024-08-27T15:43:14+00:00"
+                    }
+                ],
+                "vmAgentVersion": "2.11.1.12"
```

Comment:
This is being recognized as a secret. It can be ignored by showing in the .secret-ignore file in the pack.